proto/buildbuddy_service.proto

```diff
@@ -40,6 +41,10 @@ service BuildBuddyService {
   rpc GetInvocationOwner(invocation.GetInvocationOwnerRequest)
       returns (invocation.GetInvocationOwnerResponse);
 
+  // Archive API
+  rpc GetArchiveManifest(archive.GetArchiveManifestRequest)
```

Comment:
ubernit: this CL looks p clean but in the future it'd be nice to send the proto defs first then the service impl after

Reply:
Sorry about that, got a little excited with all of this, will split things up reasonably next time.